### PR TITLE
options: add set_info_log_level to control log verbosity

### PR DESCRIPTION
**Pull request by tkintscher**
The info log file can get quite large, and on some machines I don't need the fully verbose logging.

RocksDb provides a setting for the log verbosity; this pull request exposes it on the Rust side.

Related issues: #260 , #432

### CODE REVIEW

File: src/db_options.rs [hunk 1]
```diff
@@ -789,6 +789,18 @@ impl Default for CuckooTableOptions {
     }
 }
 
+// Verbosity of the info LOG.
+#[derive(Debug, Copy, Clone, PartialEq)]
+#[repr(i32)]
+pub enum InfoLogLevel {
```
[the remark this answers is not in the document]
Comment (tkintscher) [reply src/db_options.rs:795]: You're right, I will change it.

**Comment by tkintscher** [comment]
Agreed, I've turned it into a doc-test.

File: tests/test_rocksdb_options.rs [hunk 2]
```diff
@@ -16,7 +16,7 @@ mod util;
 
 use std::{fs, io::Read as _};
 
-use rocksdb::{BlockBasedOptions, DataBlockIndexType, Options, ReadOptions, DB};
+use rocksdb::{BlockBasedOptions, DataBlockIndexType, LogLevel, Options, ReadOptions, DB};
```
Comment (aleksuss) [inline tests/test_rocksdb_options.rs:19]: It seems that `LogLevel` is unnecessary import here ?

Comment (tkintscher) [reply tests/test_rocksdb_options.rs]: My apologies, it's fixed.

**Review by stanislav-tkach — APPROVED** [review]

**Review by aleksuss — APPROVED** [review]

**Pull request merged**